google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -410,45 +410,42 @@ public void testListBlobsEmptySelectedFields() throws InterruptedException {
 
   @Test(timeout = 5000)
   public void testListBlobRequesterPays() throws InterruptedException {
-    String[] blobNames = {"test-list-blobs-empty-selected-fields-blob1",
-        "test-list-blobs-empty-selected-fields-blob2"};
-    BlobInfo blob1 = BlobInfo.newBuilder(BUCKET, blobNames[0])
-        .setContentType(CONTENT_TYPE)
-        .build();
-    BlobInfo blob2 = BlobInfo.newBuilder(BUCKET, blobNames[1])
-        .setContentType(CONTENT_TYPE)
-        .build();
-    Blob remoteBlob1 = storage.create(blob1);
-    Blob remoteBlob2 = storage.create(blob2);
-    assertNotNull(remoteBlob1);
-    assertNotNull(remoteBlob2);
-
-    Page<Blob> page = storage.list(BUCKET,
-        Storage.BlobListOption.prefix("test-list-blobs-empty-selected-fields-blob"),
-        Storage.BlobListOption.fields());
+    BlobInfo blob1 =
+        BlobInfo.newBuilder(BUCKET, "test-list-blobs-empty-selected-fields-blob1")
+            .setContentType(CONTENT_TYPE)
+            .build();
+    assertNotNull(storage.create(blob1));
 
     // Test listing a Requester Pays bucket.
     Bucket remoteBucket = storage.get(BUCKET, Storage.BucketGetOption.fields(BucketField.ID));
     assertNull(remoteBucket.requesterPays());
     remoteBucket = remoteBucket.toBuilder().setRequesterPays(true).build();
     Bucket updatedBucket = storage.update(remoteBucket);
     assertTrue(updatedBucket.requesterPays());
-    String projectId = remoteStorageHelper.getOptions().getProjectId();
     try {
-      page = storage.list(BUCKET,
+      storage.list(
+          BUCKET,
           Storage.BlobListOption.prefix("test-list-blobs-empty-selected-fields-blob"),
           Storage.BlobListOption.fields(),
           Storage.BlobListOption.userProject("fakeBillingProjectId"));
       fail("Expected bad user project error.");
     } catch (StorageException e) {
       assertTrue(e.getMessage().contains("User project specified in the request is invalid"));
     }
-    while (Iterators.size(page.iterateAll().iterator()) != 2) {
+
+    String projectId = remoteStorageHelper.getOptions().getProjectId();
+    for (; ; ) {
```

Comment:
Isn't it more common to do `while(true)` in Java?

Reply:
I don't know. I like `for` because it has fewer characters, but `while` is fine with me.

Reply:
Characters, schmaracters. It increases the token count, which I feel is worse :-P